src/containers/reports/ReportsUtils.js

```diff
@@ -24,7 +24,7 @@ import {
 const compileSubjectData = (subjectData :Map, reportData :Map) => {
 
   const dobUnknown = !subjectData.hasIn([FQN.PERSON_DOB_FQN, 0]);
-  const subjectDob = subjectData.getIn([FQN.AGE_FQN, 0], '');
+  const subjectDob = subjectData.getIn([FQN.PERSON_DOB_FQN, 0], '');
```

Comment:
you can probably simplify the line above given that you're loading PERSON_DOB_FQN here -- you can just move it below and set it to `dobUnknown = !!subjectDob`